snp_motion_planning/src/motion_client.py

```
@@ -0,0 +1,175 @@
+        if response.success:
+            client.get_logger().info("Received motion plan successfully!")
+            # Example: print number of points in each trajectory
+            client.get_logger().info(
+                f"Approach: {len(response.approach.points)} points"
+            )
+            client.get_logger().info(f"Process: {len(response.process.points)} points")
+            # print(response.approach.points)
+            client.get_logger().info(
+                f"Departure: {len(response.departure.points)} points"
+            )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)